Source/player.h

```diff
@@ -397,6 +406,22 @@ void LoadPlrGFX(int pnum, player_graphic gfxflag);
 void InitPlayerGFX(int pnum);
 void InitPlrGFXMem(int pnum);
 void FreePlayerGFX(int pnum);
+
+/**
+ * @brief Specifies what special logics are applied for a Animation
+ */
+enum AnimationDistributionFlags : uint8_t {
+	None = 0,
+	/*
+	* @brief ProcessAnimation will be called after NewPlrAnim (in same GameTick as NewPlrAnim)
+	*/
+	ProcessAnimationPending = 1 << 0,
+	/*
+	* @brief Delay of last Frame is ignored (for example, because only Frame and not delay is checked in game_logic)
+	*/
+	SkipsDelayOfLastFrame = 1 << 1,

```

Comment:
Why specify these as bit flag values? You aren't using combinations anywhere as far as I can see?

Reply:
After looking at `monsterdata` I saw some animations with delay that aren't the standing animation. So I thought there will be animations that needs a combination.
And I dislike a method calls like `NewAnimation(..., true, false, true, false, true)`. :smile:

But as you said, currently they are not needed.
If you want I can convert them to `enum class` and introduce the flag-logic later if we encounter the specific animation. 🙂

Reply:
I world prefer that :)

Reply:
done 🙂